pkg/parser/parser_test.go

```diff
@@ -328,6 +318,115 @@ oom_score = 0
 	)
 })
 
+var _ = Describe("Test NBContract compatibility from Json to CSE command", func() {
+	DescribeTable("for test case", func(folder string, validator outputValidator) {
+		nBCB := nbcontractv1.NewNBContractBuilder()
```

Comment:
nit: not too fond of the naming, kinda confusing, but can revisit

Reply:
With this, the description in junit.xml looks smooth to me but definitely can revisit for a better one.
An example in junit.xml
`Test NBContract compatibility from Json to CSE command for test case with empty config. Parser Should provide default values to unset fields.`